### DISCUSSION TITLE
Unobserved exceptions from `OnlineSpectatorClient.BeginPlayingInternal`

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#20522] [Q&A] [answered]
I played via really-really bad wi-fi network.
[network.log](https://github.com/ppy/osu/files/9669086/network.log)
[runtime.log](https://github.com/ppy/osu/files/9669088/runtime.log)

It should reconnect silently.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20526
